student-test/assignment1/Test_validate_username.c

```diff
@@ -18,5 +18,7 @@ void test_validate_my_username()
      * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
      * config file and my_username() functions are setup properly
      */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    const char* return_from_conf = malloc_username_from_conf_file();
+    const char* return_from_validate = my_username();
+    TEST_ASSERT_EQUAL_STRING_MESSAGE(return_from_conf,return_from_validate,"They both are not same");
```

Comment:
@Rishikesh38 what cleanup is needed for `return_from_conf` here?

Reply:
I missed the free( ) for malloc.